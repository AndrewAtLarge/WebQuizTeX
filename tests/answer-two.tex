% should give error as has two \answer's
\documentclass[hidesidemenu]{webquiz}
\title{String comparison for the answer}
\begin{document}
  \begin{question}     % a quiz question
     What is the capital of Australia?
     \answer[string]{Canberra}
     \answer[string]{Sydney}
     \whenRight  Yes, Sydney is the capital of NSW and Melbourne is the
     capital of Victoria
     \whenWrong Remember to correctly capitalise your answer!
  \end{question}
\end{document}
