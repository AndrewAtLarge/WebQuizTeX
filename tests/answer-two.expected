! Package webquiz Error: Each question can contain only one \answer command.
